Improve documentation (#236)
OCA/mis-builder: docs/usage.rst

------------------------------------------

> Check the module `mis_builder_demo <https://github.com/OCA/mis-builder/tree/10.0/mis_builder_demo>`_ to see how to create specific series of data for Committed purchases not yet invoiced.

Reviewer: This explanation about committed purchases is not really related to the Queries tab. It's probably better to mention it in a section that talks about selecting "data sources" when configuring a column.

Reviewer: This can be done in another PR of course.

Reviewer 2: in another PR!

> Example of query: we want a report showing the Total amount (HVAT) of all sales order confirmed during the current month and the previous month.

Reviewer: HVAT => Excluding VAT.

> Example of query: we want a report showing the Total amount (HVAT) of all sales order confirmed during the current month and the previous month.

Reviewer 3: ```suggestion
Example of query: we want a report showing the Total amount (Excluding VAT) of all sales order confirmed during the current month and the previous month.
```

Create a query:

* Name: select a name (it must be a python expression, so no space in the name)
* Model: look for the model on which you want to do the report. For example : Sale Order
* Field to fetch: select in the list the field from the Model that will be used in the report. For example: Untaxed Amount
* Fetched fields name: this will show the name of the field to use in the KPI expression later on. For example: amount_untaxed

> * Aggregate: choose between nothing, Sum, Max, Average or Min. Use either this aggregate field or you will need to add some aggregate expression in the KPI, because the query will give a list, not a number.

Reviewer: Use eiter this aggregate field or => If you leave this aggregate field empty

> * Aggregate: choose between nothing, Sum, Max, Average or Min. Use either this aggregate field or you will need to add some aggregate expression in the KPI, because the query will give a list, not a number.

Reviewer 2: ```suggestion
* Aggregate: choose between nothing, Sum, Max, Average or Min. If you leave this aggregate field empty, the query will give a list, not a number.
```

> * Date field: choose a date field available on the Model. For example: confirmation date of the sale order.

Reviewer: Perhaps say a word explaining what this date field will be used for, i.e. selecting records that match the date period of the columns

Reviewer 2: ```suggestion
* Date field: choose a date field available on the Model. This date is used to get the records that matched the period mentioned in the query. For example: confirmation date of the sale order.
```

* Domain: this is optional. Use a domain (as definded in classic Odoo). For example: [("partner_id.country_id.code","=","US")]

.. image:: _static/images/query_1.png
   :width: 1800

In the KPI expression, you can now use the fields of the queries.
For example:

> sum([s.amount_untaxed for s in sales_order_amount])

Reviewer: In this part, it would be nice to show both cases: this one which works when the aggregate field has been left empty, and show an example where aggregate has been set.

.. image:: _static/images/query_2.png
   :width: 1800

In a reporting instance, the result is as follows:

.. image:: _static/images/query_3.png
   :width: 1800

> 

Reviewer 2: Another case is to use the internal way with: the name of the query and the fetched field name.

<img width="1122" alt="query_3" src="https://user-images.githubusercontent.com/6325713/65949791-28ca5f00-e43d-11e9-9e0d-d870ab1276ad.png">

<img width="1126" alt="query_4" src="https://user-images.githubusercontent.com/6325713/65949792-28ca5f00-e43d-11e9-885f-7a6a4f459417.png">

Analytic Filters
----------------
Analytic accounts is often used in budget and actual versions to follow-up the costs and expenses of a project.

> If you need to activate the management of analytic accounts, go to Accounting module ‣ Configuration ‣ Settings and enable the Budget management feature.

Reviewer: Is it not "Analytic accounting" instead of "Budget management" ?

> If you need to activate the management of analytic accounts, go to Accounting module ‣ Configuration ‣ Settings and enable the Budget management feature.

Reviewer 2: ```suggestion
If you need to activate the management of analytic accounts, go to Invoicing module ‣ Configuration ‣ Settings and enable the Analytic Accounting.
```
<img width="442" alt="analytic-accounts" src="https://user-images.githubusercontent.com/6325713/65950012-89599c00-e43d-11e9-92ae-bcf711b7d6c0.png">
